controllers/etcdpeer_controller.go

```diff
@@ -646,23 +656,35 @@ func (r *EtcdPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 		TLS: tlsConfig,
 	}
 
+	// Always requeue after ten seconds, as we don't watch on the membership list. So we don't auto-detect changes made
+	// to the etcd membership API.
+	// TODO(#76) Implement custom watch on etcd membership API, and remove this `requeueAfter`
+	result := ctrl.Result{RequeueAfter: time.Second * 10}
+
+	var healthy bool
+
 	if c, err := r.Etcd.New(etcdConfig); err != nil {
 		log.Error(err, "Unable to connect to etcd")
+		healthy = false
+		r.Recorder.Event(&peer, "Warning", "MemberConnectionFailure", "Cannot connect to etcd member")
+		if r.lastHealthy[peer.Name].IsZero() {
+			log.V(2).Info("Peer has never been healthy")
```

Comment:
Why the (2) level, don't think we use this

Reply:
This is a very good point. #22 changes it so it's always logged